e2ev2/src/config.js

```diff
@@ -8,7 +8,13 @@ export_({projectName})
 const usernames = ['puppeteer-tester-7@fake-research-aou.org']
 export_({usernames})
 
-const urlRoot = () => `https://${process.env.UI_HOSTNAME}`
+const urlRoot = () => {
+  assert(
+    process.env.UI_HOSTNAME,
+    'UI_HOSTNAME not defined. Try: export UI_HOSTNAME=all-of-us-workbench-test.appspot.com'
```

Comment:
Would it be worth using a variable for "all-of-us-workbench-test.appspot.com" in case it ever changes?

Reply:
I imagine we will want to run these against other environments, so I anticipate that this will likely need to be a variable, but I would not add that complexity before it is needed, since I never trust those predictions.